compiler/src/dotty/tools/dotc/transform/ContextFunctionResults.scala

```diff
@@ -85,14 +85,11 @@ object ContextFunctionResults:
       else
         val defn.ContextFunctionType(params, resTpe, erasedParams) = tp: @unchecked
         val rest = contextParamCount(resTpe, crCount - 1)
-        if erasedParams.contains(true) then erasedParams.count(_ == false) + rest else params.length + rest
+        // TODO use mt.nonErasedParamCount
+        if erasedParams.contains(true) then erasedParams.count(_ == false) + rest else params.length + rest // TODO use mt.nonErasedParamCount
```

Comment:
There are two TODOs here

Reply:
I will remove that code soon with other parts of #18305. I'll remove both todos then.